versions/linea-devnet.env

```diff
@@ -1,5 +1,5 @@
-LINEA_BESU_TAR_GZ=24.12-delivery40
-LINEA_SEQUENCER_PLUGIN_VERSION=0.8.0-rc8.3
+LINEA_BESU_TAR_GZ=24.12-delivery41
+LINEA_SEQUENCER_PLUGIN_VERSION=0.8.0-rc8.4
```

Comment:
Can we update mainnet one in the same change? Mainnet is used in linea-monorepo atm

Reply:
update also mainnet

Reply:
Thanks!